.eslintrc.js

```diff
@@ -124,7 +124,7 @@ module.exports = {
     'no-underscore-dangle': 'off',
     'class-methods-use-this': 'off',
     'consistent-return': ['off', { treatUndefinedAsUnspecified: true }],
-    'spaced-comment': ['error', 'always', { markers: ['/////'] }],
```

Comment:
There are a number of lint issues in this pr. Want to fix now, or later? Also, what is this change trying to do? I think the current rule is saying you can only add a bunch of slashes if you do one of our "set" comment dividers - ie, //////// JS PATHS //////

Reply:
I needed this to work:
```/// <reference types="chrome"/>```

The config I changed was found in a stackoverflow solution.

Reply:
I think you need to change to ['/////','///']

Reply:
Done